lib/vanagon/engine/pooler.rb

```diff
@@ -36,12 +37,17 @@ def load_token
       end
 
       def token_from_file
-        token_file = File.expand_path("~/.vanagon-token")
+        vanagon_token_file = File.expand_path("~/.vanagon-token")
+        vmfloaty_token_file = File.expand_path("~/.vmfloaty.yml")
 
-        if File.exist?(token_file)
-          return File.open(token_file).read.chomp
+        if File.exist?(vanagon_token_file)
+          puts "Reading vmpooler token from: #{vanagon_token_file}"
+          return File.open(vanagon_token_file).read.chomp
```

Comment:
I know this code predates this update, but this could probably be simplified with 

```ruby
File.read(vanagon_token_file).chomp
```

Also, should we consider using `#strip` instead of `#chomp` here? I'm not sure if we care about leading whitespace here. Again, just sort of thinking out loud about old code -- don't block on these questions.